info.json

```diff
@@ -1,10 +1,10 @@
 {
   "name": "Vehicle Wagon",
-  "version": "1.1.3",
-  "factorio_version": "0.15",
+  "version": "1.2.0",
+  "factorio_version": "0.16",
   "title": "Vehicle Wagon",
   "author": "Supercheese",
   "homepage": "https://forums.factorio.com/viewtopic.php?f=93&t=31489",
-  "dependencies": ["base >= 0.15.0", "? trucks"],
+  "dependencies": ["base >= 0.16.0", "? trucks"],

```

Comment:
add soft dependency for AII vehicles?

Reply:
Fixed now - I added optional dependencies for all of the vehicle packages even though strictly speaking, it's only aai-programmable-vehicles that causes the error fixed here.  

That said, if I get enough time later I'll see about adding in actual graphics for all of them.